hyperparameter_tuning/huggingface_multiclass_text_classification_20_newsgroups/hpo_huggingface_text_classification_20_newsgroups.ipynb

```diff
@@ -0,0 +1,1363 @@
+{
```

Comment:
<p>adding some detailed on this will be helpful for the users, including a warning saying that, if there is any existing directory with the "20_newsgroups_bulk" name this cell will remove all the contents from that. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2931/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>